tools/bq-queryplan-sankey/readme.md

```diff
@@ -0,0 +1,111 @@
+# Sankey visualization for BigQuery query plan
+
+## Overview
+[BigQuery](https://cloud.google.com/bigquery/) embeds diagnostic query plan information within its query jobs. This utility provides [Sankey diagram](https://en.wikipedia.org/wiki/Sankey_diagram) visualization for various stages in the query plan. The tool is based on [Sankey library](https://github.com/d3/d3-sankey) from [d3js](https://d3js.org/).
+
+## Setup
+Download or clone this repo into a workspace directory.
+
+```bash
+git clone [repo url]
+cd [workspace dir]/bq-queryplan-sankey
+```
+
+Next, to see the query plan in a Sankey digram, we need to download a job information file. You can use below command to get the query plan file for a given job id from BigQuery.  The file needs to be copied into `bqshow` directory.
+
+To use an example query plan, use `sample.json` provided in `bqshow` directory.
+
+```bash
+cd [workspace dir]/bq-queryplan-sankey/bqshow
+bq ls --jobs [project id] // list jobs in a project
+bq show --format=prettyjson --job [job id] > sample.json	// sample.json file has query plan information
+```
+
+Start the server to visualize query plan as Sankey diagrams. You'll need a web server to run this utility. Any number of options are good as this tool is currently serving static content only. 
+
+### Using Python
+With Python `3.x` to run the server on port `9090`:
+```bash
+cd [workspace dir]/bq-queryplan-sankey
+python3 -m http.server 9090
+```
+
+With Python `2.x` to run the server on port `9090`:
```

Comment:
Is this really necessary? I think this is easier to maintain if we stick to a single python version. I'd remove the python 2 section.

Reply:
Removed Python 2.x instructions.